modin/pandas/resample.py

```diff
@@ -61,6 +63,11 @@ def __init__(
             "origin": origin,
             "offset": offset,
         }
+        if PandasCompatVersion.CURRENT != PandasCompatVersion.LATEST:
```

Comment:
Should this stuff be in `modin._compat`? Actually, I think we do not want the API layer to know about compat versions.

Reply:
That's a good note, I don't think I have a good way to implement that in this case, though - the only thing which comes to mind is to make same `ResampleCompat` classes the way we do for others, but this feels like a lot of grunt work to just get rid of two lines. I'm open for suggestions and other opinions, cc @modin-project/modin-core

Reply:
> the only thing which comes to mind is to make same ResampleCompat classes the way we do for others

This is fine by me.

Reply:
reworked (and fixed a bug I believe)

Reply:
I still see that the API layer knows about `modin._compat`. Please file an issue where we will discuss the concept in order not to block the changes here.

Reply:
https://github.com/modin-project/modin/issues/5086

thanks for reminding!